### PR TITLE
create slice-node, closes: #24

### PR DESCRIPTION
**Pull request by HVish**

### CODE REVIEW

**Comment by LosYear** [comment]
Thanks, @HVish! 

Right now user is unable to use `Slice` component and to fix that you should:
1. add export in `src/index.ts` 
2. export your slice renderer in `renderers/index.ts`

Also, I had some problems with specifying `x` and `y` props on component. Would you mind fixing this stuff?

**Comment by HVish** [comment]
Sure! I will fix it.

**Comment by LosYear** [comment]
Hi, @HVish! 
Sorry for not replying sooner.

To make it clear, please consider running following example:
```jsx
import * as React from 'react';
import { Page, View, Text, Slice } from 'react-figma';

export const App = () => {
    return (
        <Page name="New page" isCurrent>
            <View>
                <View style={{ width: 200, height: 100, backgroundColor: '#dd55aa' }} />
                <Text style={{ color: '#ffffff'}}>text</Text>
                <Slice style={{width: 200, height: 100, top:0, left:0, position: 'absolute'}} />
            </View>
        </Page>
    );
};
```
If you try running it, you can see that absolute positioning is not working. This is what I get on the screen
image

File: src/components/slice/Slice.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import { StyleOf, BaseNodeProps, SceneNodeProps, LayoutProps, ExportProps } from '../../types';
+import {
+    LayoutStyleProperties,
+    transformLayoutStyleProperties
+} from '../../styleTransformers/transformLayoutStyleProperties';
+import { useYogaLayout } from '../../hooks/useYogaLayout';
+import { StyleSheet } from '../..';
+
+export interface SliceProps extends BaseNodeProps, SceneNodeProps, LayoutProps, ExportProps {
+    style?: StyleOf<LayoutStyleProperties>;
```
Comment (LosYear) [inline src/components/slice/Slice.tsx:11]: I guess it should be `style?: StyleOf<LayoutStyleProperties & YogaStyleProperties>`;

**Comment by HVish** [comment]
Hi @LosYear 

I ran into this problem of `x`, `y` co-ordinates while I was implementing and testing it. I was trying to debug the issue but could not fix it.

So far now I have noticed that for first render it renders normally as expected, then in next render cycle the co-ordinates are overwritten. After applying some breakpoints I found out that the `style` prop is lost in somewhere before reaching the `applyStyleToYogaNode()`.

Can you help me with this?

**Comment by ilyalesik** [comment]
Just a `saveStyleMixin(sliceNode)(props);` was needed, I added it. Thanks, everyone!

**Pull request merged**